report/src/main/kotlin/com/hartwig/actin/report/pdf/chapters/OtherTrialMatchingResultsChapter.kt

```diff
@@ -14,7 +14,7 @@ import com.hartwig.actin.report.trial.TrialsProvider
 import com.itextpdf.kernel.geom.PageSize
 import com.itextpdf.layout.Document
 
-class TrialMatchingChapter(
+class OtherTrialMatchingResultsChapter(
```

Comment:
I don't really like the name and also don't think its correct with every permutation of the environment config but I can't think of a better suggestion now